protocols/epic.js

```diff
@@ -126,7 +126,21 @@ export default class Epic extends Core {
       throw new Error('Server not found')
     }
 
-    state.name = desiredServer.attributes.CUSTOMSERVERNAME_s
+    if (
+      desiredServer.attributes.NAME_s !== undefined &&
+      desiredServer.attributes.NAME_s !== null &&
+      desiredServer.attributes.NAME_s !== ''
+    ) {
+      state.name = desiredServer.attributes.NAME_s;
+    } else if (
+      desiredServer.attributes.CUSTOMSERVERNAME_s !== undefined &&
+      desiredServer.attributes.CUSTOMSERVERNAME_s !== null &&
+      desiredServer.attributes.CUSTOMSERVERNAME_s !== ''
+    ) {
+      state.name = desiredServer.attributes.CUSTOMSERVERNAME_s;
+    } else {
+      state.name = '';
+    }
```

Comment:
Maybe we can have something like this?

```js
const name = desiredServer.attributes.NAME_s
const customName = desiredServer.attributes.CUSTOMSERVERNAME_s

if (name && name !== '') {
  state.name = name
} else if (customName && customName !== '') {
  state.name = customName
} else {
  state.name = ''
}
```

Reply:
Sure, fine with me. 
My JS is very old knowledge and therefore rusty, is just test and give an idea, good execution and looking beatiful, that i will leave up to you guys 😎

Reply:
Updated it